docs/workflows/partnership-pages.md

```diff
@@ -2,21 +2,28 @@
 
 ## NU (WES):
 
-- Create a new page
-- Add content from copy
+- Go to [NU Pre-production Site](https://nu-edu-preprod.go-vip.co/wp-admin/)
+- [Add or edit a partnership page](https://nu-edu-preprod.go-vip.co/wp-admin/edit.php?post_type=partnership)
+- [Compress any given PDFs](https://smallpdf.com/)
+- [Compress any given images and logos if necessary](https://compressor.io/)
+- Beautify the name of the files. Example: `NU_Tulare_County_Flyer_06.2021.pdf` to `Tulare_County_Flyer.pdf`
+- Add the content given by copy
 - Update `Page Attributes`:
-    - Parent: `Workfoce Education Solutions`
+    - Parent: `No Parent`
     - Template: `Standard Hero Sidebar`
+    - Order: `0`

```

Comment:
Pretty sure this one's unnecessary. Unless someone has a reason we use this, please remove.